docs/bbe-guidelines/bbe-guidelines.md

```diff
@@ -0,0 +1,174 @@
+# Guidelines for Writing Ballerina By Examples (BBEs)
+
+## General Best Practices
+
+1. The maximum character count per line should be 80 in .bal files. Otherwise, the lines will be wrapped in the [Ballerina](https://ballerina.io/) website reducing the readability.
```

Comment:
Was the limit 80? I recall it being 60 something.

Reply:
And the character count doesn't apply for comments in the .bal file, though it would be a good practice to keep it to 80 if possible

Reply:
Ah ignore the 2nd comment above. Can see that it's already addressed

Reply:
Counted from the below line in [1]. The character limit is 80. Reworded as follows:

"It is recommended to limit the maximum character count per line to 80 in .bal files. Otherwise, the lines get automatically wrapped in the [Ballerina website](https://ballerina.io/) reducing the readability as shown below."

"io:println("Insert to table failed: " + <string>retWithKey.detail().message);"

[1] https://github.com/ballerinax/jdbc/blob/master/jdbc-extension-examples/examples/jdbc-client-crud-operations/jdbc_client_crud_operations.bal